### ISSUE TITLE
New release plder pi image

### ISSUE THREAD

**Issue by Rocknrolldoggie** [Wizarrrr/wizarr#102]
Hi it's me again. Thank you so much for the work you put in. I am getting a "no matching manifest for Linux/arm/v7 in the manifest list entries". Am I doing something incorrect ? Using the normal rocker compose method. (Cli) gave me the same error.

**Comment by mtthidoteu** [comment]
Ah, weird, I apologise, I wasn't able to test it as I don't have any arm/v7 architecture, but I assumed it would work as the image built same as the others.

Try adding `--platform=linux/arm/v7` or `platform: linux/arm/v7` ? 

I'll be honest, I am not very confident, but give it a try!

**Comment by Rocknrolldoggie** [comment]
Hi. Don’t apologize you trying and I would love to help to help others.
Let’s try it.

**Comment by Rocknrolldoggie** [comment]
New strang error even trying to run it the original way that gave the previous error

Error response from daemon: Head "https://ghcr.io/v2/wizarrrr/wizarr/manifests/latest": Get "https://ghcr.io/token?scope=repository%3Awizarrrr%2Fwizarr%3Apull&service=ghcr.io": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by sander1095** [comment]
The issue that @Rocknrolldoggie is reporting seems to be local, because after working around a different issue, everything works fine.

@mtthidoteu this will be relevant for you:

## The `latest` tag is set to an older version
I am running a pi 3b+. I think the `latest` tag should still be updated to AT LEAST point to the `2.1.3` version to make the repo nice and compatible with pi 3b+'s without requiring us to specify a specific version of wizarr:

```
# The latest tag points to an outdated version where a pi 3b+ (armv7 arch) isn't supported yet:
> docker pull ghcr.io/wizarrrr/wizarr
Using default tag: latest
latest: Pulling from wizarrrr/wizarr
no matching manifest for linux/arm/v7 in the manifest list entries
```

When I try to pull `2.1.3`, it does work. The latest tag is [still on the 2.1.2 tag](https://github.com/wizarrrr/wizarr/pkgs/container/wizarr), so by default wizarr won't work on pi's. 

```
# Tag 2.1.3 does work on pi 3b+'s:
> docker pull ghcr.io/wizarrrr/wizarr:2.1.3
2.1.3: Pulling from wizarrrr/wizarr
6fb81ff47bd6: Pull complete
2ccb1e226af4: Pull complete
3e91ff0e619c: Pull complete
22f299025f60: Pull complete
0790cfb3385b: Pull complete
64a19b0af7de: Pull complete
6ced051d9e27: Pull complete
4f4fb700ef54: Pull complete
31257913791b: Pull complete
9bbaf307d338: Pull complete
a6349cbc9ac6: Pull complete
d603adab6b5a: Pull complete
Digest: sha256:8d43aa6543ccde76f58188e06eaaddbf8555f8b9a1e19d894d6e6b9b4baccead
Status: Downloaded newer image for ghcr.io/wizarrrr/wizarr:2.1.3
ghcr.io/wizarrrr/wizarr:2.1.3
```

Proof:

![image](https://user-images.githubusercontent.com/7312681/225750460-6219007c-9126-4800-b864-1a9f038fea04.png)


## Trying to fix @Rocknrolldoggie's issue
Try to follow this setup:

- Try to have a `docker-compose.yml` like this:

```---
version: "3.8"
services:
  wizarr:
    container_name: wizarr
    image: ghcr.io/wizarrrr/wizarr:2.1.3 # TODO: Remove the `:2.1.3` text when latest tag points to this version 
    #user: 1000:1000 #Optional but recommended, sets the user uid that Wizarr will run with
    ports:
      - 5690:5690
    volumes:
      - /mnt/usb_hdd/wizarr/config:/data/database # TODO: Change the left part of the `:` to your path.
    environment:
      - APP_URL=https://wizarr.yourdomain.com # TODO: Change this URL to where you will host and access this application
      - DISABLE_BUILTIN_AUTH=false #Set to true ONLY if you are using another auth provider (Authelia, Authentik, etc)
      - TZ=Europe/Amsterdam #Set your timezone here
```
- Take a look at your `volumes`. We will clean up any items in there so it will be clean when we run `wizarr` for the first time. To clean this up, do the following (replace my path with the path that you have in your `volumes` section):
  - `cd /mnt/usb_hdd/wizarr/config`
  - [`rm -rf ..?* .[!.]* *`](https://unix.stackexchange.com/a/77313/565625)
- now run `docker-compose up -d` in the directory where you saved the `docker-compose.yml`

If you have any more issues, let us know!

**Comment by Rocknrolldoggie** [comment]
Hellooooo

It works it works !!!! Well until the setup page at least I will fill everything in when I get home from work. I had to redo the docker without the UID tag though as it caused permission errors I could not resolve not matter what permissions I handed out.

I am soooo grateful to both of you & will add this to every sub reddit I know will enjoy it & let them know about your kofi page too. End month will get you coffee but with exchange rate from South Africa its not a mid month offering. 

Sander to you too all the best with your studies etc and indebted to you.
